custom_commands/suppressedNotificationsTrueCheck.js

```diff
@@ -1,70 +1,66 @@
 const axios = require('axios');
 
 module.exports = {
-    command: async function () {
-
-        const getToken = async () => {
-            const response = await axios.post(
-                'https://qa2.development.tidepool.org/auth/login',
-                '',
-                {
-                    headers: {
-                        'Authorization': 'Basic YnJpYW4rdXBsb2FkNzg2d2ViMTMyN2NsaW5pYzM1YUB0aWRlcG9vbC5vcmc6dGlkZXBvb2w=',
-                        'Content-Type': 'application/x-www-form-urlencoded'
-                    }
-                })
-            return response.headers['x-tidepool-session-token']
-        }
+  command: async function (clinicianUsername, clinicianPassword, environment) {
+    const auth = btoa(`${clinicianUsername}:${clinicianPassword}`);
+    const getToken = async () => {
+      const response = await axios.post(
+        `${environment}/auth/login`,
+        '',
+        {
+          headers: {
+            Authorization: `Basic ${auth}`,
+            'Content-Type': 'application/x-www-form-urlencoded',
+          },
+        },
+      );
+      return response.headers['x-tidepool-session-token'];
+    };
 
+    const clinicData = async () => {
+      const token = await getToken();
+      response = await axios.get('https://qa2.development.tidepool.org/v1/clinicians/4ae71760b6/clinics', {
```

Comment:
**issue (if-minor):** clinic ID `4ae71760b6` is only on QA2 - we need to handle other envs as well

Reply:
clinic id moved to global environment variables

see commit e138f9274a6bf9b2d17620d2a10426b1651c4fb2